### PR TITLE
Buffer Occupancy - Trace Type addition

### PR DESCRIPTION
**Pull request by rsivakolundu**
Added trace type for buffer occupancy and related text.

### CODE REVIEW

**Comment by jalemon** [comment]
Hi Ramesh,

This gives Bit 12-23 as Undefined, but then assigns Bit 22 for Checksum
Complement. Did you mean for Bit 12-21 to be Undefined and Bit 23 to
be Checksum
Complement?

Thanks, John

**Comment by mickeyspiegel** [comment]
I merged the namespace edits pull request, which fixes the confusion around bits 11-23. Please merge from master.

**Comment by rsivakolundu** [comment]
Oops.

Yes. I will correct it.

-Ramesh

File: drafts/draft-ietf-ippm-ioam-data.xml [hunk 1]
```diff
@@ -854,8 +857,7 @@ The trace option data MUST be 4-octet aligned:
 
                       <t>Not add any node data fields to the packet, even for
                       the IOAM-Trace-Type bits defined above.</t>
-                    </list></t>
```
Comment (mickeyspiegel) [inline drafts/draft-ietf-ippm-ioam-data.xml]: This line should not be deleted.

**Pull request merged**